code/game/objects/structures/watercloset.dm

```diff
@@ -414,6 +414,16 @@
 		user << "<span class='warning'>Someone's already washing here!</span>"
 		return
 
+	if(istype(O, /obj/item/weapon/reagent_containers/food/snacks/monkeycube))
+		if (/obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped)
+			user << "<span class='notice'>You need to unwrap [src] first!</span>"
+			return
+		else
+			user << "<span class='notice'>You place [src] under a stream of water...</span>"
+			user.drop_item()
+			loc = get_turf(O)

```

Comment:
Same for ```src``` used in the output stream.
Generally, this PR is simply code moving for now and needs to be cleaned up.